agent/consul/gateways/controller_gateways.go

```diff
@@ -0,0 +1,32 @@
+package gateways
+
+import (
+	"context"
+	"github.com/hashicorp/consul/agent/consul/controller"
```

Comment:
Looks like this needs a `goimports` run. I can commit that in a moment

Reply:
Thank you! I'll make sure I check on that before opening the PR next time.